src/server/CMakeLists.txt

```diff
@@ -47,6 +52,10 @@ add_library(dragonfly_lib engine_shard_set.cc channel_store.cc
             acl/user.cc acl/user_registry.cc acl/acl_family.cc
             acl/validator.cc acl/helpers.cc)
 
+if (DF_ENABLE_MEMORY_TRACKING)
+  target_compile_definitions(dragonfly_lib PRIVATE DFLY_ENABLE_MEMORY_TRACKING)
```

Comment:
is it ok that you add this define to all source files? I thought only one include on dfly_main.cc should suffice.
We already change compile flags for this file we can similarly change definitions there.

Reply:
nm